.github/workflows/JuliaNightly.yml

```diff
@@ -0,0 +1,31 @@
+name: JuliaNightly
+# Nightly Scheduled Julia Nightly Run
+on:
+  push:
+    branches: [master]
+    tags: ["*"]
+  pull_request:
```

Comment:
I removed the daily scheduled runs, since we don't notify anyone here, so running these on a daily basis is probably rather pointless.

Reply:
good call :)